grakn-core/src/main/java/ai/grakn/exception/GraqlQueryException.java

```diff
@@ -19,12 +19,20 @@
 package ai.grakn.exception;
 
 /**
- * Thrown when a grakn validation rule is broken. {@see ai.grakn.core.implementation.Validator} for more details
+ * <p>
+ *     Graql Query Exception
+ * </p>
  *
- * @author Filipe Teixeira
+ * <p>
+ *     Occurs when the query is syntactically correct but semantically incorrect.
+ *     For example limiting the results of a query -1
+ * </p>
+ *
+ * @author fppt
  */
-public class GraknValidationException extends GraphRuntimeException{
-    public GraknValidationException(String error) {
+//TODO: Felix to integrate with Graql
+public class GraqlQueryException extends GraknException{
+    public GraqlQueryException(String error) {
```

Comment:
Should this Exception take in the query which had the exception so that it can be used in the message?